.github/workflows/watch-lintdiff.yaml

```
@@ -0,0 +1,21 @@
+# Use ~ to sort the workflow to the bottom of the list
+name: "~Watch - LintDiff"
+
+on:
+  # check_suite is preferred over check_run to avoid triggering on all check
+  # runs. In some cases, check_run must be used in testing environments.
+  check_suite:
+    types: completed
+
+  workflow_run:
+    types: completed
+    workflows: 
+      - "\\[TEST-IGNORE\\] Swagger LintDiff"
+
+jobs:
+  LintDiff:
+    name: Lint Diff
```

```suggestion
  WatchLintDiff:
    name: Watch LintDiff
```

Rename to clarify this isn't actually LintDiff, but a "LintDiff Watcher".